images/hook/entrypoint.sh

```diff
@@ -1,25 +1,24 @@
 #!/bin/sh
-set -e
+set -xe
 
 # note that rig does not take explicit changeset ID
 # taking it from the environment variables
 
 echo "--> Assuming changeset from the environment: $RIG_CHANGESET"
 if [ $1 = "install" ]; then
     echo "--> Creating Log Forwarder ConfigMap"
+    # apply is used here because log-forwarder could be created by gravity
```

Comment:
is the one created by gravity in the resource system? If it does, wouldn't this have the effect of overriding user provided config?